Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
Why haven't you used the Dockerfile generated by the operator-sdk 1.0.0? It is using multi stage build. Use that one.